src/main/groovy/jp/co/soramitsu/devops/docker/DockerPlugin.groovy

```diff
@@ -269,6 +270,8 @@ class DockerPlugin implements Plugin<Project> {
             return 'openjdk:12-jdk-oracle'
         } else if (javaVersion == 13) {
             return 'openjdk:13'
+        } else if (javaVersion == 17) {
+            return 'openjdk:17-jdk-slim'
```

Comment:
It'd be better to use JRE instead of JDK.

Reply:
@Mingela it was copied from images above. How do you thing?

Reply:
doesn't matter a lot till it works, but I assume jdk may be needed to build a project 🤔